addons/api/addon/abilities/account.js

```diff
@@ -11,6 +11,14 @@ import ModelAbility from './model';
 export default class AccountAbility extends ModelAbility {
   // =permissions
 
+  /**
+   * Only "known" account types may be read.
+   * @type {boolean}
+   */
+  get canRead() {
+    return !this.model.isUnknown && this.hasAuthorizedAction('read');
+  }
```

Comment:
Should we repeat this pattern for update and delete as well, to future-proof the UI?

Reply:
Great point, i'll update this!